### ISSUE TITLE
Implement AO Landing Page

### ISSUE THREAD

**Issue by porta-antiporta** [18F/fec-eregs#196]
Per the mockup at: https://invis.io/AR7U3Y662#/167711252_AOs_Landing

Need to develop the landing page:

- [x] Search lands the user in the AO search only page
- [x] No facet support
- [x] Include AO Overview content
- [x] Include Requesting AO content
- [x] Add copy to the legal resources page for AO
- [ ] *Should* Pending AOs
- [ ] *Should* Recent AOs

**Comment by emileighoutlaw** [comment]
Is there a way for me to see this without an InVision account? I don't have one :/

**Comment by emileighoutlaw** [comment]
Per our meeting, we've got approved content for two pages:
- Legal resources landing page (https://beta.fec.gov/legal-resources/)
- On the advisory opinion subpage  (https://beta.fec.gov/legal-resources/advisory-opinions)

**Legal resources landing page content:**

> ## Advisory opinions
Advisory opinions are official Commission responses to questions about how federal campaign finance law applies to specific, factual situations.

> This archive contains advisory opinions from 1975 to the present. It also contains documents related to advisory opinions — such as requests, drafts and public comments — from 1990 to the present.

> Learn how to [request an advisory opinion or submit a public comment]() on an advisory opinion.(link to http://www.fec.gov/pages/brochures/ao.shtml)

> [Explore advisory opinions]()(link to AO subpage)



**For the advisory opinion subpage:**

> ## Advisory opinions
Advisory opinions are official Commission responses to questions about how federal campaign finance law applies to specific, factual situations.

> This archive contains advisory opinions from 1975 to the present. It also contains documents related to advisory opinions — such as requests, drafts and public comments — from 1990 to the present.

> Learn how to [request an advisory opinion or submit a public comment]() on an advisory opinion.(link to http://www.fec.gov/pages/brochures/ao.shtml)

 _Pending advisory opinions and recent advisory opinions to follow here_.

cc @amykort

**Comment by porta-antiporta** [comment]
@emileighoutlaw Regarding the landing page: https://fec-dev-proxy.18f.gov/legal-resources/

For regulations, there's an explicit `Get Started` button to send the user to the Regulations subpage.  In the example above, however, It uses linked text to send the user to the Advisory opinions subpage.  From a consistency perspective, should we change the `Get Started` button to text as well for Regulations?  If so should that text be "Explore 11 CFR".?

cc: @jenniferthibault @edmullen

**Comment by porta-antiporta** [comment]
PR created to update legal resources landing page: https://github.com/18F/fec-cms/pull/423.  Will update once I get answer to ^^

**Comment by porta-antiporta** [comment]
@emileighoutlaw on the legal resources landing page, do we want the link on how to request an advisory opinion?  I *think* that may have been a copy and paste error.

**Comment by emileighoutlaw** [comment]
Hi Porta!

### Buttons
I think my preference is that both the regulations button and the advisory opinion button say `Explore ...`

I just issued  a commit to your PR making that fix :)

### Link
I waffled about the link to the brochure on the legal resources home. I don't know if users will know that they can find out that information if it's only on the advisory opinion landing page, which they access via a link that says `Explore advisory opinions`. 

How about we leave off the link for now (to limit how many clickable options users have) and trying to suss out an answer about whether folks can find that info in testing?

**Comment by porta-antiporta** [comment]
@adborden @annalee In looking at creating the landing page.  

I believe that the appropriate URL for the landing page is `~/legal/advisory-opinions` instead of just `~/advisory-opinions`.  This is a bit different than regulations where the "landing page" is `~\regulations`.  However, I feel regulations is more of the snowflake as it puts us on a separate application.

If you agree, the route to `~/legal/advisory-opinions` is already in place.  However, that URL always assumes that the path to that is through a search (i.e. show me all the results for that search type).   My proposed approach is to modify `legal-doc-search-results.html` to be able to handle both when a search query is passed or when there is no search query.  (So the page could likely be renamed `legal-doc-specific.html` or something to that effect).

Part of what is driving this proposal is that by keeping the same page, when we eventually support facets etc, we wouldn't need to duplicate the layouts between a search results page vs. a landing page.

You good with this approach?

**Comment by adborden** [comment]
I would recommending keeping the landing page separate from the advisory opinions search page. It gets complicated when one page has multiple functions. The shared pieces between the pages, like the search sidebar, can be implemented as a macro, and then reused between the pages.

When I created `legal-doc-search-results.html`, the idea was it would serve as the search results page for all of `regulations`, `advisory-opinions`, and `murs`, but even after adding regulations that goal is getting challenging. Might copy the file, and name it specific to advisory opinions if that's the direction we end up going.

On the url, I'm in agreement that most everything should be under `/legal`. As a side note, while in development the url looks correct, on cloud foundry, the url will actually be `/data/legal` which is documented [in this issue](https://github.com/18F/openFEC-web-app/issues/1193).

**Comment by porta-antiporta** [comment]
@adborden Completed a draft the layout of the landing page.  https://github.com/18F/openFEC-web-app/pull/1394.  Tried to follow existing patterns but could be missing something.  Following up on seeing if API does support returning `pending` and `latest` AOs; but appreciate a review of what's already there.

**Comment by adborden** [comment]
Btw, here is a viewable link to the AO invision doc https://gsa.invisionapp.com/share/AR7U3Y662#/screens/167711252_AOs_Landing

**Comment by AmyKort** [comment]
Thanks for sharing the link! I'm not sure I'm in the right thread for these comments, but Cheryl took a look and asked whether you'd had good results with users on the "dates" search. She writes:

 I'd be interested to see what people think of that being on the initial search page.  That is part of our current advanced search system.   I think it would be more helpful to have a search field for AO Number.  I could put that in the "search terms" box, but I suspect I will not only get that AO but everything else that references it.  I think more often than not, people either want to search by keyword/phrase, requestor, or AO Number (and for this one, they don't want to have to sift through other things that cite to that AO number).  

I might also suggest changing the title of "Latest Advisory Opinions" to "Most Recent Advisory Opinions"

**Comment by AmyKort** [comment]
Also, another comment about wording:

 Change "Requesting Advisory Opinions" to "Request an Advisory Opinion" or "How to Request an Advisory Opinion?"

**Comment by emileighoutlaw** [comment]
Hello there! Before we discuss, I'm resurfacing the real content we decided on for the Advisory Opinion subpage, (as opposed to what's in the InVision doc that doesn't align quite right):

> ## Advisory opinions
Advisory opinions are official Commission responses to questions about how federal campaign finance law applies to specific, factual situations.

> This archive contains advisory opinions from 1975 to the present. It also contains documents related to advisory opinions — such as requests, drafts and public comments — from 1990 to the present.

> Learn how to [request an advisory opinion or submit a public comment]() on an advisory opinion.(link to http://www.fec.gov/pages/brochures/ao.shtml)

> **Pending advisory opinion requests**
- list here

> **Latest advisory opinions**
- list here

So off the bat, I think the real content addresses your concern about the `Request` language, @AmyKort, because it's called out a little differently (as an inline link). 

Can you tell me a little more about wanting to change `Latest` to `Most recent`? Latest is a pretty well-established pattern online for the this type of thing, and I think our users will recognize it and appreciate the consistency. Do you have a different take?

**Comment by AmyKort** [comment]
The content you just posted looks great. If "latest" is a well-established pattern, then let's go with that.

Thanks for the clarification!

**Comment by anthonygarvan** [comment]
@AmyKort currently, if you search for either the AO number (eg., 1986-06) or the requestor name (eg., Fund for America's Future), the relevant AO shows up as the first hit. We have made some progress on the backend incorporating dates but that won't be usable until we roll out faceting.

**Comment by AmyKort** [comment]
Sounds good. I think Cheryl's comment might have been more that she wasn't sure whether the date range search option was worth the amount of emphasis it gets on the page. But maybe our external users would really benefit from searching for AOs within a certain date range . . .

**Comment by porta-antiporta** [comment]
@AmyKort and Cheryl,  some of the background behind adding the facets that are there is to mimic functionality on the legacy site.  Also, specifically to dates, one of the feedback on MURs (which may or may not apply to advisory opinions) is that some of usability test folks indicated they wanted to be able to search by date "given the commissioners change every so often and thus some of the tendencies of the MUR outcomes".

That being said, we aren't going to have facets in this initial release.  In addition for it (no facets) being an MVP; it also gives us the opportunity to perform usability tests with the simple search box to surface what _facets are truly the most important to users_.

**TL:DR;** while the mockup in the description does include facets like date;  they won't be supported in this first release and we will research which facets are the most important to prioritize.

**Comment by AmyKort** [comment]
Awesome! Thank you for explaining all of that!

**Comment by adborden** [comment]
@emileighoutlaw apologies, I missed [this comment](https://github.com/18F/fec-eregs/issues/196#issuecomment-236966215) about removing the link, and I think I added it back. Let me know how this looks and I can remove it

https://fec-stage-proxy.18f.gov/data/legal/advisory-opinions/

> How about we leave off the link for now (to limit how many clickable options users have) and trying to suss out an answer about whether folks can find that info in testing?

**Comment by porta-antiporta** [comment]
@adborden https://github.com/18F/fec-cms/pull/435 for ^^

**Comment by emileighoutlaw** [comment]
Don't know how I missed this, but 👍
